Add simple API docs (#40)
MetaMask/metamask-onboarding: README.md

## Usage

[See _§ Onboarding Library_ on the MetaMask Docs website for examples.](https://docs.metamask.io/guide/onboarding-library.html)

## API

Assuming `import MetaMaskOnboarding from '@metamask/onboarding'`, the following API is available.

### `MetaMaskOnboarding.isMetaMaskInstalled()`

> Returns the installation status of MetaMask.

Reviewer: We might want to warn users that we aren't providing any guarantee here, e.g.
```suggestion
Returns `true` if a MetaMask-like provider is detected, or `false` otherwise. Note that we don't provide any guarantee that this is correct, as non-MetaMask wallets can disguise themselves as MetaMask.
```

### `new MetaMaskOnboarding()`

The constructor accepts an optional options bag with the following:

| Option | Description |
| :-- | :-- |
| `forwarderOrigin` | **Optional**, defaults to `'https://fwd.metamask.io'` |

> | `forwarderMode` | **Optional**, defaults to `MetaMaskOnboarding.FORWARDER_MODE.INJECT` |

Reviewer: It'd be nice to show what the other option is. It looks like `FORWARDER_MODE` isn't documented.

> ### `MetaMaskOnboarding#startOnboarding()`

Reviewer: I don't love using the `#` to indicate an instance method - personally I was not familiar with this syntax before we spoke about it, and I wouldn't have known what it meant.

> Starts onboarding by opening the MetaMask download page and the Onboarding forwarder.

Reviewer: We should avoid talking about the forwarder if possible, or the internal mechanics of this library in general, and instead focus on the purpose of the method.

```suggestion
Starts onboarding by opening the MetaMask download page and waiting for MetaMask to be installed. Once the MetaMask extension installation is detected, a message will be sent to MetaMask to register the current site as the onboarding initiator.
```

### `MetaMaskOnboarding#stopOnboarding()`

> Stops onboarding registration, including removing the injected forwarder (if any).

Reviewer: ```suggestion
Stops onboarding registration, including removing the injected `iframe` (if any).
```
